with-firebase-saml-login/app.config.js

```diff
@@ -0,0 +1,9 @@
+export default {
```

Comment:
Use app.json

Reply:
I just modified the .config.js from form the other firebase style example, but for a JSON I'm not sure, how does [this](https://github.com/expo/examples/pull/290/commits/0e626a147e93a70ea60f5bb4e747b7479c258e36) look?